examples/apikeys.md

```diff
@@ -0,0 +1,42 @@
+# apikeys endpoint
+
+
+#### Create API Key
+
+Since your creating an API keys, it's likely you don't have an existing one use. Using environment variables is an easy
```

Comment:
Some grammar changes:
`Since you're creating an API key, it's likely you don't have an existing one to use`

Reply:
updated